tests/setup_windows_spatialite.py

```diff
@@ -24,3 +25,25 @@
 
 with open(file, "w") as stream:
     yaml.dump(parameters, stream, default_flow_style=False)
+
+if 'WINDOWS' in platform.platform().upper():
+    # We now set sqlite. Only needed in thge windows server in Github
+    plats = {'x86': 'https://sqlite.org/2020/sqlite-dll-win32-x86-3320100.zip',
+             'x64': 'https://sqlite.org/2020/sqlite-dll-win64-x64-3320100.zip'}
+
+    outfolder = 'C:/'
+    zip_path64 = join(outfolder, "sqlite-dll-win64-x64-3320100.zip")
+    urllib.request.urlretrieve(plats['x64'], zip_path64)
+
+    zip_path86 = join(outfolder, "sqlite-dll-win32-x86-3320100.zip")
+    urllib.request.urlretrieve(plats['x86'], zip_path86)
+
+    root = 'C:/hostedtoolcache/windows/Python/'
```

Comment:
is this for running in github actions?
maybe there is another env var you could use to distinguish running in automated environment... otherwise this will trigger for anyone who  tries to run your test setup right?

Reply:
This is just for Github Actions and it is **not** triggered when running tests. It is only on that folder for organization purposes